### PR TITLE
Adds Vehicles/Roads M113 LAV25

### PR DESCRIPTION
**Pull request by AC2ServerAccount**
## About the Pull Request

adds roads for the upcoming vehicles
This is an image

## Why It's Good For The Game

roads

## Changelog

:cl:
add: roads
/:cl:

### CODE REVIEW

**Comment by Pyrodes** [comment]
Real

**Comment by AGasCanister** [comment]
that fucking sprite, incredibly based
IKneel

File: maps/site53/site53-2.dmm [hunk 1]
```diff
@@ -30713,6 +30710,9 @@
 	},
 /turf/simulated/floor/tiled/monotile,
 /area/site53/lowertrams/hub)
+"qVm" = (
```
Comment (Vivalas) [inline maps/site53/site53-2.dmm]: This all looks good. I uh, I really have no idea why this map edit is here, it seems to change nothing. Please remove it.

**Review by Vivalas — CHANGES_REQUESTED** [review]

File: maps/site53/site53-2.dmm (hunk 1 above)
Comment (User-U-U) [reply maps/site53/site53-2.dmm:30713]: I've tried it won't go away

Comment (Vivalas) [reply maps/site53/site53-2.dmm]: Take the file from the commit before and just paste it into your repo. You can view the history of the repository at specific points in history, like a wiki

Comment (Vivalas) [reply maps/site53/site53-2.dmm]: My main concern is this fucking up and causing conflicts for people with actual map PRs.

Comment (User-U-U) [reply maps/site53/site53-2.dmm:30713]: It only effects one tile

Comment (Vivalas) [reply maps/site53/site53-2.dmm]: It doesn't matter, it's an unnecessary change. It's my job to enforce standards, as pedantic as they may seem at times. Hell I'll do it myself if you want, but you'll have to wait

Comment (User-U-U) [reply maps/site53/site53-2.dmm:30713]: I don't have the ability to fix it right now, and my next pr is messing with that area

Comment (ItsVyzo) [reply maps/site53/site53-2.dmm]: > I don't have the ability to fix it right now, and my next pr is messing with that area

download the codebase as a zip and get the area from there using SDMM

**Review by Vivalas — APPROVED** [review]
you have been blessed by me remote force-pushing to your fork to fix your own PR 👁️

File: code/game/turfs/flooring/flooring.dm [hunk 2]
```diff
@@ -413,3 +413,21 @@
...
+
+/decl/flooring/road
+	icon = 'icons/turf/roads.dmi'
+	icon_base = "road"
+
+/decl/flooring/road/empty
+	icon_base = "road_empty"
+
+/decl/flooring/road/corner
+	icon_base = "road_corner"
```
Comment (Vivalas) [inline code/game/turfs/flooring/flooring.dm:427]: this state doesn't exist in the icon

Comment (Vivalas) [reply code/game/turfs/flooring/flooring.dm:427]: road_arrow does, however

**Review by Vivalas — CHANGES_REQUESTED** [review]
that being said, there are still issues with the code proper

**Comment by Vizoplez** [comment]
why is the sprite so compressed

**Comment by AC2ServerAccount** [comment]
what u mean i drew it myself in plaint

File: code/game/turfs/flooring/flooring.dm (hunk 2 above)
Comment (AC2ServerAccount) [reply code/game/turfs/flooring/flooring.dm:427]: This ones really confusing me, I'm working on it, I think the sprites are somewhere but I am still trying to find them in the other code.

**Comment by AC2ServerAccount** [comment]
gonna remake this later.

**Pull request closed without merging**